### PR TITLE
Container link in browse v2 sidebar

### PR DESCRIPTION
**Pull request by joshuaeilers**
Changes
* Adds a container link to the browse v2 sidebar
* Analytics when clicking the container link
* Delay the sidebar tooltips by 2s
* Removes flex gap and replaces with padding because some button components inherently are padding their contained icons so it was doubling their padding with the extra gap.
* Shrink the ellipsis maxWidth when we're dealing with a container, would be really nice to use a dynamic width ellipsis here but it doesn't work and I've wrestled it a few times with no success.
* Fixes a UI glitch in the error component when we're more nested.

Screenshot 2023-06-26 at 1 33 23 PM

## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: datahub-web-react/src/app/analytics/event.ts [hunk 1]
```diff
@@ -23,6 +23,7 @@ export enum EventType {
     BrowseV2ToggleSidebarEvent,
     BrowseV2ToggleNodeEvent,
     BrowseV2SelectNodeEvent,
+    BrowseV2ClickContainerLinkEvent,
```
Comment (joshuaeilers) [inline datahub-web-react/src/app/analytics/event.ts:26]: Would like to rename this to BrowseV2ContainerLinkClickEvent

**Review by chriscollins3456 — APPROVED** [review]
great stuff! I have a suggestion that is a simple change and then a question about whether 2 seconds feels like a long time.

File: datahub-web-react/src/app/search/ToggleSidebarButton.tsx [hunk 2]
```diff
@@ -40,7 +40,7 @@ const ToggleSidebarButton = ({ isOpen, onClick }: Props) => {
     if (pauseTooltip) return button;
 
     return (
-        <Tooltip title={title} placement={placement} arrowPointAtCenter>
+        <Tooltip title={title} placement={placement} arrowPointAtCenter mouseEnterDelay={2}>
```
[the remark this answers is not in the document]
Comment (joshuaeilers) [reply datahub-web-react/src/app/search/ToggleSidebarButton.tsx:43]: Felt good to me! Rather lower it?

[the remark this answers is not in the document]
Comment (joshuaeilers) [reply datahub-web-react/src/app/search/ToggleSidebarButton.tsx:43]: We can try 1s

[the remark this answers is not in the document]
Comment (chriscollins3456) [reply datahub-web-react/src/app/search/ToggleSidebarButton.tsx]: ehh i'll say up to you. maybe 1s or 1.5s or something to meet in the middle

**Review by chriscollins3456 — APPROVED** [review]
nice!

**Pull request merged**